script/citest.sh

```diff
@@ -1,7 +1,5 @@
 jsonsyn
 grunt jshint
-#eslint src/** ; exit 0
-#eslint data/** ; exit 0
 grunt nsp
 grunt eslint
-yarn run import area=Hessen
+yarn run import area=Germany
```

Comment:
Please remember that this is so much load for the overpass api and it strongly discourages doing such downloads automatically(like in CI). It would have been better if you talked to me before doing this.

I'm not willing to merge the pr with this change.